tests/virt_autotest/virt_utils.pm

```diff
@@ -303,10 +303,10 @@ sub generate_guest_asset_name {
 
     #get build number
     my $build_num;
-    #for clone job, get build number from SCC proxy which is set in Media
-    if (get_var('CASEDIR')) {
-        get_var('SCC_URL') =~ /^http.*all-([\d\.]*)\.proxy\.*/;
-        $build_num = $1;
+    # for a clone job, the setting BUILD must be set as it was in its original job, for example, BUILD=98.1
+    # or the job will not know in which build guest assets should be download or uploaded
+    if (get_var('CASEDIR') and get_var('BUILD') =~ /\d+.?\d*/) {
```

Comment:
Similar here?  Exclude CASEDIR check and impact?

Reply:
This is a different issue. In current test, Without BUILD set explicitly in a clone job command line, fg. `openqa-clone-custom-git-refspec -c " --apikey xxx --apisecret xxx" https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/xxx http://openqa.suse.de/tests/11077448 BUILD=98.1`, test will fail due to unable to find a guest assets. My change here is to abort test ahead of actually downloading assets and tell users why their tests failed .

Reply:
What are you trying to match here ?
```
get_var('BUILD') =~ /\d+.?\d*/)
```
Need ```!``` to go with the opposite ?

Reply:
Looking into... and do more verify run.. thanks.

Reply:
corrected. new verification run: https://openqa.suse.de/tests/11153679#step/download_guest_assets/8